src/common/appId.js

```diff
@@ -35,3 +33,68 @@ export const oauthLogin = (done = () => 0) => {
         done();
     }
 };
+
+export const getAppIdFallback = () => getStorage('config.app_id') || getDefaultEndpoint().appId;
```

Comment:
Can we have a neat `getEndpoint()` function that returns both `appId` and `apiUrl` and can be used with a spread syntax:

```
LiveApi({
  websocket,
  ...getEndpoint(),
})
```

Reply:
There is a function which does something similar. We have to modify and it requires lot of change and can be used only in one place.